apps/swirl-docs/src/components/CodePreview/CodePreviewRequestString.tsx

```diff
@@ -0,0 +1,18 @@
+import { useCodePreviewContext } from "./CodePreviewContext";
+
+export function Request() {
```

Comment:
**suggestion:** The naming is a bit generic. Maybe something like "APIEndpointHeader"?